src/models/disassemblyoutput.cpp

```diff
@@ -17,28 +17,46 @@
 #include <QProcess>
 #include <QStandardPaths>
 
+#include "formattingutils.h"
+
 namespace {
 Q_LOGGING_CATEGORY(disassemblyoutput, "hotspot.disassemblyoutput")
 
-bool canVisualizeJumps(const QString& objdump)
+QString objdumpHelp(const QString& objdump)
```

Comment:
just return a `QByteArray` and use that then with `QByteArrayLiteral` - then we don't need the extra `QString` conversion